AnkiDroid/src/test/java/com/ichi2/anki/CardBrowserTest.kt

```diff
@@ -924,4 +927,53 @@ class CardBrowserTest : RobolectricTest() {
         advanceRobolectricUiLooper()
         assertThat(cardBrowser.mCards.size(), equalTo(3))
     }
+
+    @Test
+    fun `column spinner positions are set to 0 if no preferences exist`() = runBlocking {
+        // GIVEN: No shared preferences exist for display column selections
+        val sharedPrefs = targetContext.sharedPrefs()
+        sharedPrefs.edit().apply {
+            remove(CardBrowser.DISPLAY_COLUMN_1_KEY)
+            remove(CardBrowser.DISPLAY_COLUMN_2_KEY)
+            apply()
+        }
```

Comment:
Kotlin defines an `edit { }` extension which is a little less code

ditto with the other test

```suggestion
        getSharedPrefs().edit { 
            remove(CardBrowser.DISPLAY_COLUMN_1_KEY)
            remove(CardBrowser.DISPLAY_COLUMN_2_KEY)
        }
```

Reply:
@david-allison, thank you for the suggestion. I went ahead and cleaned it up a bit.